### ISSUE TITLE
onBlur is not called when year/month mode is activated

### ISSUE THREAD

**Issue by trickreich** [YouCanBookMe/react-datetime#535]
### I'm Submitting a ...

```
[X] Bug report
[ ] Feature request
[ ] Support request
```

### Steps to Reproduce
Configure datepicker in year or month mode.

### Expected Results
User selects year/month and the blur event will triggered.

### Actual Results
Blur event isn't triggerd at all when in mode month/year.

**Comment by layneanderson** [comment]
I don't think this is a bug.  The blur event isn't supposed to trigger when selecting year or month.  Selecting the year then triggers the month to be displayed, then clicking on month triggers the day to be displayed.  Only when the day is finally selected will the blur event be triggered.

**Comment by trickreich** [comment]
You have misunderstood me.

I'm talking about when the component isn't in date mode configured. Event is missing when the component is an month picker or an year picker.

**Comment by trickreich** [comment]
So can we reopen this please?

**Comment by GuiRitter** [comment]
I too came across this when using `dateFormat={'MM/YY'}`. It seems that `onBlur` is called for the `<input/>` but not for the `<div class="rdtPicker"/>`.
